src/base/b-virtual-scroll/README.md

```diff
@@ -0,0 +1,31 @@
+# base/b-virtual-scroll
+
+This module provides a component to render a large number of elements with Vue.
```

Comment:
This module provides a component to render component sequences with the support of lazy loading and dynamically updating.
This component can be very efficient if you need to render a good amount of elements.